src/shared/constants.ts

```diff
@@ -109,6 +109,7 @@ export const AlterationTypeConstants = {
 
 export const DataTypeConstants = {
     DISCRETE: 'DISCRETE',
+    DISCRETE_LONG: 'DISCRETE_LONG',
```

Comment:
we decided again this, right?

Reply:
Yes, totally right, removed in [515fec4](https://github.com/cBioPortal/cbioportal-frontend/pull/4418/commits/515fec4546c8becb2c90691991dc6ee44973ffe3)